documents/final-submission-document/chapters/1_introduction.tex

```diff
@@ -74,4 +77,16 @@ \subsection{IETF Internet-Draft: Using XMPP for Security Information Exchange}
 
 \section{Motivation}
 In this section, we legitimate this thesis and explain the value and applicability of our proposed solution.
-% TODO: From AVT - use and introduce the term "Broker" \gls{broker}
+
+\subsection{Present situation}
+The IETF standard draft \emph{Using \gls{xmpp} for Security Information Exchange} \cite{ietf-mile-xmpp-grid-05} as summarised in section \ref{sec:ietf-internet-draft-using-xmpp-for-security-information-exchange} defines a protocol to exchange security-relevant information between endpoints.
+The draft was created by the Managed Incident Lightweight Exchange (MILE) Working Group to support computer and network security incident management.
+
+To demonstrate the viability of the draft, a HSR-Rapid Prototype was developed in November~2017~\cite{xmpp-grid-prototype}.
+
+\subsection{Problem and Solution}
+Currently, there exists no implementation of the \gls{xmpp} grid draft management functionality which is ready for production use in terms of usability and security.
+
+To solve this problem, a graphical interface with bindings to a suitable \gls{broker} must be proposed and implemented.
+It must permit network administrators to comfortably manage and review \glspl{topic}, persisted items and \glspl{platform} as well as according access and publishing permissions of XMPP grid \glspl{platform}.
```

Comment:
This sentence is a bit long and does not work entirely (comfortably manage and review topics)...I haven't yet figured out a better way to write it.. split it up in multiple sentences?